spec/operations/event_procedures/build_total_amount_cents_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe EventProcedures::BuildTotalAmountCents, type: :operation do
+  describe ".result" do
+    it "returns a success" do
+      procedure = create(:procedure)
+      attributes = {
```

Comment:
Is there an entity in this domain that can be represented by a `precedure, urgency and room_type`?
Looks like we should have an entity that can be represented by this, looking at the code, it is not natural for me to know why we have a hash and not an entity for this